Packs/Legacy/Scripts/IPNetwork/IPNetwork.py

```diff
@@ -0,0 +1,77 @@
+# STD imports
+import ipaddress
+from typing import Dict, Tuple
+
+import demistomock as demisto
+from CommonServerPython import *
+
+# Local imports
+
+# Gather information regarding CIDR -
+#     1. Broadcast_address
+#     2. CIDR
+#     3. First_address
+#     4. Last address
+#     5. Max prefix len
+#     6. Num addresses
+#     7. Private
+#     8. Version
```

Comment:
I meant the `comment` key in the yml which acts as the script descriptions in Demisto.